documentation/cloud-manager-guide/admin/user-management/edit-user.md

```diff
@@ -0,0 +1,122 @@
+---
+keywords:
+title: Edit a User
+description: Edit a user
+---
+# Edit a User
+
+This guide outlines the required steps to edit the details, status, role, and environment assignments for a specific user in Environment Operations Center.
+
+## Getting started
+
+There are two ways to open the *Edit User* tab, either by selecting the user's name or by selecting the **Options** (**...**) menu associated with that user.
+
+To access the *Edit User* tab by directly selecting the user, select the user name from the list of users on the *Admin* *User* tab.
+
+![image description](images/edit-select-username.png)
+
+This brings you to the *Environment Access* tab in a detailed overview of the user. This view lists all the environments and associated roles assigned to the user. From the *Environment Access* tab, select the **Edit User** button to open the *Edit User* tab.
+
+![image description](images/edit-select-edituser.png)
+
+Alternatively, from the *User* tab you can select the **Options** (**...**) menu associated with the user. Select **Edit** from the dropdown menu to bypass the user detailed view and navigate directly to the *Edit User* tab.
+
+![image description](images/edit-users-options.png)
+
+The *Edit User* tab contains all of the same categories and fields as the *Create User* tab. From here you can update the user's details, status, role, and environment assignments accordingly.
+
+![image description](images/edit-user-tab.png)
+
+## Update user details
+
+From the *Edit User* tab, you can update a user's first name, last name, and email address in the *User Details* section. Enter the updated information in the associated field provided and select **Save** to update the user's details.
+
+![image description](images/edit-user-details.png)
+
+## Update user status
```

Comment:
This would be also be good to split with some H3s - one for the workflow from Options and another for the workflow from Edit User screen.